uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-vacancies.js

```diff
@@ -0,0 +1,15 @@
+var modal = require('./page');
+
+module.exports = modal.extend({
+  /**
+   * Opens More Details section
+   *
+   * @return {object}
+   */
+  showMoreDetails: function () {
+    this.chromy.click('.fieldset-title');
+    this.chromy.wait(2000);
```

Comment:
Please leave the comment here, why 2000 is needed. I think there was a comment in the original file.

Reply:
There wasn't https://github.com/civicrm/civihr/blob/d613043233822228b27cd3389bfaabc4a0665a90/uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp/vacancies.js#L14-L17

I didn't author the file so I don't now why there is specific timeout

Reply:
@AkA84 Apologies, it was a similar code block, could you please add a similar comment as we have here (I suspect these are very similar blocks of code):
https://github.com/civicrm/civihr/pull/2535/files#diff-8f3c68e64102b0500aab1019270cf451R9